dallinger/heroku/clock.py

```diff
@@ -45,7 +45,24 @@ def check_db_for_missing_notifications():
     run_check(participants, config, reference_time)
 
 
+@scheduler.scheduled_job("interval", minutes=1)
+def rerecruit_when_recruiters_have_a_shortfall(recruiter=None):
+    """Some recruiters don't always recruit enough players, this checks
+    each recruiter and if that is the case re-calls the recruit method"""
+    if recruiter is None:
+        config = dallinger.config.get_config()
+        recruiter = recruiters.from_config(config)
+    if hasattr(recruiter, "counter"):
```

Comment:
This leaks a lot of detail about how the recruiter works and why it wants to be notified. How about just:
```
recruiter.ping() # or recruiter.ping(datetime.now())
``` 
and the rest is inside the implementation for `ping()`.

The base Recruiter could provide a `ping()` that's a no-op.

Reply:
Yeah, I like that a lot more.